rest/models/pom.xml

```diff
@@ -54,6 +54,7 @@
             <plugin>
                 <groupId>org.apache.servicemix.tooling</groupId>
                 <artifactId>depends-maven-plugin</artifactId>
+                <version>1.4.0</version>
```

Comment:
✏️ Patrick said that we should use pluginManagement instead of putting plugin versions in the poms of sub-projects.

Reply:
Thanks. I'll take care of that.

Reply:
It was already defined in plugin management. I guess it had to be or else Maven would have spat out an error. It's odd that Maven generated a warning about it missing a version.

Reply:
I don't see `depends-maven-plugin` in the root pom anywhere: https://github.com/connexta/ion-ingest-api/blob/6cdfc84f00e2bd4237f481edc871ae9e45aadb67/pom.xml. I'm not sure we're talking about the same thing.